src/layouts/sidebar-sidecar/sidebar-sidecar-layout.module.css

```diff
@@ -75,9 +75,9 @@ for further details.
   @media (--dev-dot-tablet-down) {
     bottom: 0;
     left: -150vw;
-    position: absolute;
-    top: 0;
+    position: absolute; /** Top position set dynamically based on scroll pos in component */
     width: 100vw;
+    padding-bottom: 70px;
```

Comment:
We needed a bit extra space to allow the whole component to be scrollable. We can fiddle with this number.

Reply:
💭 Thought, very much non-blocking and outside the scope of this PR: I think this is a fundamental issue with alert bar height mismatch. I think we might be able to remove the need for this if we tackle this other task unrelated to the contents of this PR:
- 🎟️ [[sidebar slash top nav] fix mismatch in CSS height of top bars vs reality, especially on mobile](https://app.asana.com/0/1202114367927919/1202316169578550/f)

Not sure exactly what to do in the scope of this PR - I think it meets the criteria of scroll lock. So more a random related mention than anything else 👍

Reply:
This ticket's also very relevant, and I think it'll reduce the complexity of the brittle height variable we have:

🎟️ [[alpha vs beta feedback] when should we drop the "alpha" feedback callout?](https://app.asana.com/0/1202110981600689/1202285743002882/f)

Reply:
🗒️ This is just a breadcrumb in case it comes up next week, but I'll be adding the version select (which is fixed to the bottom left side of the page, "within" the sidebar) so some tweaks might be needed on this value then.

Reply:
Sounds good. I'll link to this convo in those tickets 👌